### PR TITLE
Fetch from DB - Budget Table

### PR DESCRIPTION
**Pull request by MahmodMhamed**
Fetch budget data from db closes#55

### CODE REVIEW

File: server/src/controllers/budget.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 const { getBudgetData, addBudgetItem } = require('../models/budget');
 
 exports.fetchBudgetData = (req, res) => {
-  res.json(getBudgetData());
+  let data = [];
```
Comment (okarem) [inline server/src/controllers/budget.js:4]: data [] is not used, remove it please

File: client/src/screens/Budget/Budget.jsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ const Budget = () => {
       .get('http://localhost:5000/api/getBudget')
       .then(res => res.data)
       .then(finalRes => {
+        console.log(finalRes);
```
Comment (okarem) [inline client/src/screens/Budget/Budget.jsx:29]: console.log(finalRes); 
remove please, it done it's job

File: server/src/models/budget.js [hunk 3]
```diff
@@ -1,31 +1,24 @@
...
 
-
- 
-
 exports.addBudgetItem = async (itemName, quantity, price, category) => {
-  //I added the call to the DB insert query inside the function that inserts data to Json file, 
-  //so it is called once the API fitched from frontend
-  //send userId hardcoded until we send it with the cookie
-  const result = insertToBudget(1,itemName, quantity, price, category,(err, res) => {
+  const result = insertToBudget(1, itemName, quantity, price, category, (err, res) => {
```
Comment (okarem) [inline server/src/models/budget.js:15]: remove `const result = ` you don't use it anywhere

File: server/src/models/budget.js [hunk 4]
```diff
@@ -1,31 +1,24 @@
...
     if (err) {
       return err;
     }
     return res;
   });
- 
-  
+
   return new Promise((resolve, reject) => {
```
Comment (okarem) [inline server/src/models/budget.js:22]: your function ALWAYS retuns line 19 or line or line 17
everything aftrer line 20 is useless in this case

**Review by okarem — CHANGES_REQUESTED** [review]

**Comment by MahmodMhamed** [comment]
Duplicate #59

**Pull request closed without merging**